pkg/crd/generator/generator.go

```diff
@@ -32,6 +31,7 @@ import (
 	"sigs.k8s.io/controller-tools/pkg/internal/codegen"
 	"sigs.k8s.io/controller-tools/pkg/internal/codegen/parse"
 	"sigs.k8s.io/controller-tools/pkg/util"
+	"sigs.k8s.io/yaml"
```

Comment:
this block is generally intended to be for other packages within this module, so `"sigs.k8s.io/yaml"` still belongs in the second block